services/storage/service.go

```diff
@@ -0,0 +1,73 @@
+package storage
+
+import (
+	"time"
+
+	"github.com/influxdata/influxdb/services/meta"
+	"github.com/influxdata/influxdb/tsdb"
+	"github.com/uber-go/zap"
+)
+
+// Service manages the listener and handler for an HTTP endpoint.
+type Service struct {
+	addr           string
+	yarpc          *yarpcServer
+	loggingEnabled bool
+	logger         zap.Logger
+
+	Store      *Store
+	TSDBStore  *tsdb.Store
+	MetaClient interface {
+		Database(name string) *meta.DatabaseInfo
```

Comment:
Neither of these are called. I assume that's just on the `TODO` list? 😄

Reply:
The service does not use them, but the type is passed to the `Store`